frontend/packages/dev-console/integration-tests/support/step-definitions/addFlow/resource-quota-warning.ts

```diff
@@ -0,0 +1,69 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+import { detailsPage } from '@console/cypress-integration-tests/views/details-page';
+import { listPage } from '@console/cypress-integration-tests/views/list-page';
+import { modal } from '@console/cypress-integration-tests/views/modal';
+import * as yamlView from '@console/cypress-integration-tests/views/yaml-editor';
+import { devNavigationMenu } from '@console/dev-console/integration-tests/support/constants';
+import {
+  navigateTo,
+  createGitWorkloadIfNotExistsOnTopologyPage,
+  topologyHelper,
+  app,
+} from '@console/dev-console/integration-tests/support/pages';
+
+const deteleResourceQuota = () => {
+  detailsPage.isLoaded();
+  detailsPage.clickPageActionFromDropdown('Delete ResourceQuota');
+  modal.shouldBeOpened();
+  modal.submit();
+  modal.shouldBeClosed();
+};
+
+Given('user has created workload with resource type deployment', () => {
+  createGitWorkloadIfNotExistsOnTopologyPage(
+    'https://github.com/sclorg/nodejs-ex.git',
+    'ex-node-js',
+    'deployment',
+    'nodejs-ex-git-app',
+  );
+  topologyHelper.verifyWorkloadInTopologyPage('ex-node-js');
+});
+
+Given('user has created two resource quotas {string} and {string} using {string} file', () => {
+  cy.exec(`oc apply -f testData/resource-quota/resource-quota.yaml`);
+  app.waitForDocumentLoad();
+});
```

Comment:
Edit the feature file according to this
```suggestion
Given('user has created two resource quotas using {string} file', (yamlLocation) => {
  cy.exec(`oc apply -f ${yamlLocation}`);
  app.waitForDocumentLoad();
});
```

Reply:
Updated.